php/commands/help.php

```diff
@@ -118,6 +118,12 @@ private static function pass_through_pager( $out ) {
 			$pager = Utils\is_windows() ? 'more' : 'less -r';
 		}
 
+		// For Windows 7 need to set code page to something other than Unicode (65001) to get around "Not enough memory." error with `more.com` on PHP 7.1+.
+		if ( 'more' === $pager && defined( 'PHP_WINDOWS_VERSION_MAJOR' ) && PHP_WINDOWS_VERSION_MAJOR < 10 && function_exists( 'sapi_windows_cp_set' ) ) {
+			// Note will also apply to Windows 8 (see http://msdn.microsoft.com/en-us/library/windows/desktop/ms724832.aspx) but probably harmless anyway.
+			sapi_windows_cp_set( 1252 ); // `sapi_windows_cp_set()` introduced PHP 7.1. Code page 1252 is the most used so probably the most compat.
```

Comment:
Does it make sense to pick the codepage through an overrideable ENV variable? That way, impacted users can switch if needed.